### DISCUSSION TITLE
Alire index and packages

### DISCUSSION THREAD

**Discussion by krouziciorel** [alire-project/alire#1351] [Ideas]
In my tests of Alire, I had trouble installing some of the packages that are available on the project website. All after consulting the Reddit forum my problem was solved by the command:

alr index --update-all

I think other developers might experience a similar problem, so I suggest adding this command to the documentation on:

https://alire.ada.dev/docs/

**Comment by mosteo** [comment]
Thanks for the suggestion, there's already an issue to try and improve this aspect #1348 in response to your experience.

**Reply by krouziciorel** [reply]
Dear Mosteo,

Sorry for the duplicate entry, Fabien was faster... Anyway, Alire is an excellent tool that has brought Ada into the modern times, so I'm happy to help by at least reporting bugs or possible improvements to make it as pleasant to use as possible.

**Reply by mosteo** [reply]
No prob! Better safer than sorry :)
